subsys/bluetooth/mesh/proxy_srv.c

```diff
@@ -391,6 +413,9 @@ static void private_node_id_start(struct bt_mesh_subnet *sub)
 {
 #if defined(CONFIG_BT_MESH_PRIV_BEACONS)
 	sub->priv_beacon_ctx.node_id = true;
+
+	node_id.subnets++;
```

Comment:
Might this counter be desynchronized since subnetwork state isn't checked for incrementation? After decrementation it will roll over zero to max type.

Reply:
Right, need to fix this.